### ISSUE TITLE
[Request] Support setting ReplayKit (Screen Recorder) to always record at native resolution

### ISSUE THREAD

**Issue by QuanTrieuPCYT** [leminlimez/CowabungaLite#85]
On jailbroken iDevices, this behavior can be set with the `defaults write replayd RPFullResCapture -bool 1` as root in a terminal, which I believe should have a configurable file laying somewhere in the system.

**Comment by leminlimez** [comment]
My guess is that the replay kit userdefaults are stored in `/var/mobile/Library/Preferences/com.apple.replayd.plist` but I did not see a file there. I will try adding it myself and see if that works.

**Comment by QuanTrieuPCYT** [comment]
Don't know if this should work on newer devices, as I only tested this only on my jailbroken iOS 15.6 7 Plus, so looking forward for the result

**Comment by leminlimez** [comment]
can you confirm in filza that the file exists there?

**Comment by QuanTrieuPCYT** [comment]
Nope, unfortunately. I'm also doing a full filesystem search for any plists matching `replayd` but seems like it will gonna take a while

**Comment by QuanTrieuPCYT** [comment]
Well I found the file, it's called `replayd.plist`, has one entries (`RPFullResCapture` as a boolean set to `true`)

But unfortunately the file is located in the `/private/preboot/BE463051320B8A72DC83C34251A48E8CFBD1BA2653F20F3D457EBF6F632D72D7BC2F098DFB815D58B2920915927080AF/jb-m3GC1XdN/procursus/var/mobile/Library/Preferences` path which is somewhat related to the Procursus bootstrap, can you try manually creating the file in the real `Preferences` folder and see if it works

**Comment by QuanTrieuPCYT** [comment]
Update on the matter: seems like the `defaults` command is restricted to the jailbroken environment only, doing a screen record in a non-jailbroken state doesn't seem to work (still being limited at plain HD)

**Comment by QuanTrieuPCYT** [comment]
Tried manually creating both `replayd.plist` and `com.apple.replayd.plist` in the `Preferences` directory and doesn't seem to work. Maybe some inspection in the `replayd` binary could be required to find the correct configurable plist?

**Comment by leminlimez** [comment]
preboot is basically a symlink, it is just to prevent jb detection
